ts/src/alpaca.ts

```diff
@@ -552,7 +668,7 @@ export default class alpaca extends Exchange {
         const clientOrderId = this.safeString (params, 'clientOrderId', defaultClientId);
         request['client_order_id'] = clientOrderId;
         params = this.omit (params, [ 'clientOrderId' ]);
-        const order = await this.privatePostOrders (this.extend (request, params));
+        const order = await this.traderPrivatePostV2Orders (this.extend (request, params));
```

Comment:
@sc0Vu if I try to place an order in the paper trading mode I get this error, is this expected?
```
✗ n alpaca createOrder "BTC/USD" market buy 1 5 --sandbox             
2023-11-16T12:40:32.255Z
Node.js: v18.18.0
CCXT v4.1.54
alpaca.createOrder (BTC/USD, market, buy, 1, 5)
PermissionDenied alpaca {"message": "forbidden."}

---------------------------------------------------
[PermissionDenied] alpaca {"message": "forbidden."}

    at throwExactlyMatchedException  Users/cjg/Git/ccxt13/ccxt/js/src/base/Exchange.js:3309  
    at handleErrors                  Users/cjg/Git/ccxt13/ccxt/js/src/alpaca.js:831          
    at                               Users/cjg/Git/ccxt13/ccxt/js/src/base/Exchange.js:782   
    at processTicksAndRejections     node:internal/process/task_queues:95                    
    at fetch2                        Users/cjg/Git/ccxt13/ccxt/js/src/base/Exchange.js:2885  
    at request                       Users/cjg/Git/ccxt13/ccxt/js/src/base/Exchange.js:2888  
    at createOrder                   Users/cjg/Git/ccxt13/ccxt/js/src/alpaca.js:552          
    at async run                     Users/cjg/Git/ccxt13/ccxt/examples/js/cli.js:319        

alpaca {"message": "forbidden."}
```

Reply:
let me checked......

Reply:
@carlosmiei I used production key, can you create order in production api?